source_code/main_mcu/src/FILESYSTEM/custom_fs_defines.h

```diff
@@ -66,8 +66,9 @@
 #define SETTINGS_PIN_SHOWN_WHEN_BACK        14
 #define SETTINGS_UNLOCK_FEATURE_PARAM       15
 #define SETTINGS_DEVICE_TUTORIAL            16
+#define SETTING_SHOW_PIN_ON_ENTRY           17
```

Comment:
I actually had coded setting #14 for you :)

Reply:
Yeah :-) Wanted to see the whole thing!